README.md

```diff
@@ -187,3 +187,41 @@ cadenzaClient.on('editGeometry:ok', (event) => {
 
 _Note:_ Under the hood, creating a geometry is similar to editing a geometry.
 That's why the events use the `editGeometry` prefix.
+
+### Fetch data from a view
+
+<small>API: [CadenzaClient#fetchData](./classes/CadenzaClient.html#fetchData)</small>
+
+Fetch data from a workbook view in CSV format.
+
+```javascript
+const response = await cadenzaClient.fetchData('{embeddingTargetId}', { 
+  accept: 'text/csv'
+});
+
+const text = await response.text();
+...
+parsedTextArray.includes('foo');
```

Comment:
As a reader I would be very confused where `parsedTextArray` is coming from and why it would include `foo`. IMO the await two lines above is enough for an easy example